app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -30,16 +30,18 @@ def geocoder
       end
 
       def report_errors(error)
-        remapped_error = {
-          Faraday::ClientError => :bad_request,
-          Faraday::ConnectionFailed => :bad_request,
-          Faraday::TimeoutError => :bad_request,
-          ActionController::InvalidAuthenticityToken => :bad_request,
-        }[error.class] || :internal_server_error
+        remapped_error = case error
+        when Faraday::ClientError, Faraday::ConnectionFailed, Faraday::TimeoutError,
+          ActionController::InvalidAuthenticityToken
```

Comment:
I find this sort of multiple-one-line, multiple-multi-line mixture to be difficult to scan. Rubocop doesn't seem particularly concerned with how it's formatted, but maybe some variation of one-per-line?

```suggestion
        when Faraday::ClientError,
             Faraday::ConnectionFailed,
             Faraday::TimeoutError,
             ActionController::InvalidAuthenticityToken
```

Reply:
much better idea, thx!